### PR TITLE
[Doc] Fixed some minor syntax issues

### PR DESCRIPTION
**Pull request by javiereguiluz**
Just some minor fixes found while reviewing the docs. Cheers!

### CODE REVIEW

File: docs/cookbook/recipe_custom_action.rst [hunk 1]
```diff
@@ -111,7 +111,7 @@ to implement a ``clone`` action::
         }
     }
 
-If you want to add the current filter parameters to the redirect url you can add them to the `generateUrl` method::
+If you want to add the current filter parameters to the redirect url you can add them to the ``generateUrl`` method::
```
Comment (VincentLanglet) [inline docs/cookbook/recipe_custom_action.rst:114]: @OskarStark Is there a linter rule for this ?

Comment (phansys) [inline docs/cookbook/recipe_custom_action.rst:114]:
```suggestion
If you want to add the current filter parameters to the redirect url you can add them to the ``generateUrl()`` method::
```

File: docs/reference/advanced_configuration.rst [hunk 2]
```diff
@@ -253,7 +253,7 @@ ACL
 ^^^
 
 Though the route linked by a menu may be protected the Tab Menu will not automatically check the ACl for you.
-The link will still appear unless you manually check it using the `hasAccess` method::
+The link will still appear unless you manually check it using the ``hasAccess`` method::
```
Comment (phansys) [inline docs/reference/advanced_configuration.rst:256]:
```suggestion
The link will still appear unless you manually check it using the ``hasAccess()`` method::
```

**Review by phansys — CHANGES_REQUESTED** [review]

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
Thank you @javiereguiluz

**Pull request merged**